### ISSUE TITLE
GQL: GraphQL Playground Not Loading Schemas

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#29109]
### Problem Statement
The GraphQL playground is not loading the schemas because it needs authentication now.

### Steps to Reproduce
https://github.com/dotCMS/core/assets/751424/15f5e479-f55c-44a2-aba2-cf99290a127b

### Acceptance Criteria
- [ ] The GraphQL playground should successfully load the schemas
- [ ] It should not show any errors

### External Links
- This is the [playground we're using,](https://github.com/graphql/graphiql) maybe we need a new one?

### Assumptions & Initiation Needs
- Here is Altair GQL Client [some other insights](https://github.com/dotCMS/core/assets/751424/dfc57bcd-4bdf-45c3-abbf-0b043d7eb5af) that I found
- Postman doesn't work, [here is the video](https://github.com/dotCMS/core/assets/751424/9f53df26-8561-4774-85b0-4c09540192ae)
- Not sure if we are in an older version of GQL spec because Altair shows this [error](https://github.com/dotCMS/core/assets/751424/3b9b666f-96b1-4264-8dd1-f6462f88215f)

### Quality Assurance Notes & Workarounds
- To work around the authentication issue, manually provide the authentication token in the GraphQL playground or clients to load the schemas.

**Comment by imolorhe** [comment]
FYI Altair allows for lots of customizations if you'd also like to make it look branded

**Comment by fabrizzio-dotCMS** [comment]
@fmontes is the schema expected to load at all times even when no authorization has been granted?

**Comment by fmontes** [comment]
@fabrizzio-dotCMS no, it used to be that way and it got changed because security concerns.

https://www.apollographql.com/docs/technotes/TN0021-graph-security/#turn-off-introspection-in-production

So now the problem is that our playground stop working and some other clients are not working either.

I am not sure how the GQL endpoint should be authorized, @dsilvam told me cookie, but I also tried bearer token and it work in some GQL clients.
